documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -289,6 +289,20 @@
               description: 'Prometheus %(prometheusName)s has dropped {{ printf "%%.0f" $value }} targets because some targets exceeded the configured body_size_limit.' % $._config,
             },
           },
+          {
+            alert: 'PrometheusScrapeSampleLimitHit',
+            expr: |||
+              increase(prometheus_target_scrapes_exceeded_sample_limit_total{%(prometheusSelector)s}[5m]) > 0
+            ||| % $._config,
+            'for': '15m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'Prometheus has failed scrapes that have exceeded the configured sample limit.',
+              description: 'Prometheus %(prometheusName)s has failed {{ printf "%%.0f" $value }} scrapes because some targets exceeded the configured sample_limit.' % $._config,
```

Comment:
I realize that the description here is better than the one for the previous alert (`PrometheusScrapeBodySizeLimitHit`) because it correctly refers to scrapes, not to targets. Still, it doesn't mention the range. How about

```
'Prometheus %(prometheusName)s has failed {{ printf "%%.0f" $value }} scrapes in the last 5m because some targets exceeded the configured sample_limit.'
```

And while you are on it, could you change the description in line 289 to

```
'Prometheus %(prometheusName)s has failed {{ printf "%%.0f" $value }} scrapes in the last 5m because some targets exceeded the configured body_size_limit.'
```
?

Thank you.

Reply:
Thanks, the suggested descriptions definitely make sense.